mlrun/model_monitoring/db/tsdb/timescaledb/timescaledb_schema.py

```diff
@@ -283,12 +283,17 @@ def _get_records_query(
         desc: Optional[bool] = None,
         use_pre_aggregates: bool = True,
         group_by: Optional[list[str]] = None,
+        timestamp_column: Optional[str] = None,
     ) -> str:
-        """Build query to get records from the table or its pre-aggregates."""
+        """Build query to get records from the table or its pre-aggregates.
+
+        :param timestamp_column: Optional timestamp column to use for time filtering.
+                                If not provided, uses the table's default time_column.
```

Comment:
Just to keep consistency, if we provide docstring about params we should provide for all of the params and not just one. However, in this case I would remove that docstring and maybe add a comment before L296, saying 
`# If timestamp_column not provided, use the table's default time_column` 
```suggestion

```